zuul.d/projects.yaml

```diff
@@ -7,6 +7,7 @@
     github-check:
       jobs:
         - ci-framework-crc-podified-edpm-deployment
+        - ci-framework-crc-podified-edpm-baremetal
```

Comment:
please add it to /ci/templates/project.yaml - else it will be dropped whenever we create a new role.